testtools/content.py

```diff
@@ -163,61 +161,52 @@ def __init__(self, stack_lines, prefix_content="", postfix_content=""):
     def _stack_lines_to_unicode(self, stack_lines):
         """Converts a list of pre-processed stack lines into a unicode string.
         """
+        msg_lines = traceback.format_list(stack_lines)
```

Comment:
It's not. See the comment. I will do another patch to address setup.py and cleanup the existing (before this patch) and introduced by this patch conditionals. But it may require dropping features, not sure yet.